tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_report.py

```diff
@@ -0,0 +1,56 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+from marionette_driver import expected, By, Wait
+from gaiatest import GaiaTestCase
+from gaiatest.apps.base import Base
+from gaiatest.apps.messages.app import Messages
+
+
+class TestSmsReport(GaiaTestCase):
+
+    def test_sms_report(self):
+        
+        time_test_began = time.time()
+        _text_message_content = "Automated Test %s" % str(time_test_began)
+        _test_phone_number = self.environment.phone_numbers[0]
```

Comment:
I don't think these variables should need to be underscored, in my opinion, but it's not a big dal.